doc/sql_guide.md

```diff
@@ -0,0 +1,291 @@
+# Test Objective
+Structured Query Language (SQL), SQL is an ANSI (American National Standards Institute) standard computer language. Its functions include data query, data manipulation, data definition and data control. It is a general relational database language with strong functions. It is now the standard language for relational databases. (such as SELECT, UPDATE, DELETE, INSERT, WHERE and etc)   
```

Comment:
Replace with:

SQL is basic but important feature of one RDBMS product. CUBRID as  a comprehensive relational database management system supports not only standard SQL but also native extension.
In order to test SQL feature, we designed two suites SQL and MEDIUM which include all kinds of DDL, DML and so on.
Typically, SQL and MEDIUM perform tests via JDBC interface on both Linux and Windows platform as regression test.  And we also introduce SQL_BY_CCI suite which execute SQL test cases via CCI interface. In this guide, I would introduce SQL, MEDIUM and SQL_BY_CCI tests.